examples/CURL/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-curl)
+
+hunter_add_package(CURL)
+find_package(CURL REQUIRED)
```

Comment:
We have to use `find_package(CURL CONFIG REQUIRED)` to load `CURLConfig.cmake` instead of `FindCURL.cmake`